packages/cli/src/config/extensions/github.ts

```diff
@@ -72,6 +74,23 @@ export async function cloneFromGit(
   }
 }
 
+function parseGitHubRepo(source: string): { owner: string; repo: string } {
+  // The source should be "owner/repo" or a full GitHub URL.
+  const parts = source.split('/');
+  if (!source.includes('://') && parts.length !== 2) {
```

Comment:
This could be a followup but we should probably just properly parse this as a URI and pull out the path segments, and then check that we get two.

Or, just simplify it and force exactly owner/repo format and don't allow the URI syntax.

Reply:
yes def-- let's do in a fast-follow